packages/presentation/lib/screen/home/widgets/home_tab_bar.dart

```diff
@@ -41,40 +42,44 @@ class _HomeTabBarState extends State<HomeTabBar> {
 
   @override
   Widget build(BuildContext context) {
-    return TabBar(
-      controller: widget.tabController,
-      splashBorderRadius: BorderRadius.circular(Dimens.size16),
-      indicator: BoxDecoration(
-        color: AppColorsDark.primaryColor,
-        borderRadius: BorderRadius.circular(Dimens.size16),
-      ),
-      tabs: <Widget>[
-        Tab(
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              !isIconActive ? const Icon(Icons.play_circle) : const SizedBox(),
-              const SizedBox(width: Dimens.size6),
-              Text(
-                SM.current.nowShowing,
-                style: AppTextStyles.sfProMedium14px,
-              ),
-            ],
-          ),
+    final width = MediaQuery.of(context).size.width;
+    return SizedBox(
+      width: Responsive.isDesktop(context) ? Dimens.size400 : width,
+      child: TabBar(
+        controller: widget.tabController,
+        splashBorderRadius: BorderRadius.circular(Dimens.size16),
+        indicator: BoxDecoration(
+          color: AppColorsDark.primaryColor,
+          borderRadius: BorderRadius.circular(Dimens.size16),
         ),
-        Tab(
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              const SizedBox(width: Dimens.size6),
-              Text(
-                SM.current.comingSoon,
-                style: AppTextStyles.sfProMedium14px,
-              ),
-            ],
+        tabs: <Widget>[
+          Tab(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                !isIconActive ? const Icon(Icons.play_circle) : const SizedBox(),
```

Comment:
just write if( !isIconActive) const Icon(Icons.play_circle)

Reply:
why it's better then my way?

Reply:
agree with Alex. it allows u to write less code